dpks/quant_matrix.py

```diff
@@ -438,29 +419,45 @@ def compare_groups(
 
         return self
 
-    def impute(self, method: str, **kwargs: int) -> QuantMatrix:
+    def classify(
+        self,
+        classifier,
+        shap_algorithm: str = "auto",
+        scale: bool = True,
+        rfe_step: int = 1,
+        rfe_min_features_to_select: int = 1,
+    ) -> QuantMatrix:
+        le = LabelEncoder()
+        Y = le.fit_transform(self.quantitative_data.var["group"].values)
+        X = self.quantitative_data.X.copy().transpose()
+        X = np.nan_to_num(X, copy=True, nan=0.0)
+        if scale:
+            X = StandardScaler().fit_transform(X)
+        clf = Classifier(classifier=classifier, shap_algorithm=shap_algorithm)
```

Comment:
I think we should store the trained classifier and the scaler as attributes to the quant matrix, that way it can be used to classify new data or do many other things without it being lost when this function runs out of scope.

Just something like:

```
if scale:
    self.scaler = StandardScaler()
    X = self.scaler.fit_transform(X)

self.clf = Classifier(...)

```